projects/schematics/src/shared/utils/file-utils.ts

```diff
@@ -732,6 +738,95 @@ export function insertCommentAboveIdentifier(
   return changes;
 }
 
+export function getImportDeclarations(
+  source: ts.SourceFile,
+  importPath: string
+): ts.ImportDeclaration[] {
+  const imports = getSourceNodes(source).filter(
+    node => node.kind === ts.SyntaxKind.ImportDeclaration
+  );
+  if (!imports) {
```

Comment:
I believe `imports` will always be defined, right?

Reply:
Yep